identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources_zh_CN.properties

```diff
@@ -139,6 +139,7 @@ Invalid.reset.link=此链接已过期。
 Set.Password=设置密码
 Continue=继续
 Please.enter.valid.email=请输入有效的电子邮件地址。
+Please.enter.valid.input=请输入有效的条目。
```

Comment:
chinese and japanses does not seems to work when we use with idea IDE. Hence try it using vs code.

Reply:
Used vscode to put these lines. Anything to change?

Reply:
Ack